third_party/mkl_dnn/mkldnn.BUILD

```diff
@@ -64,7 +65,12 @@ cc_library(
         "src/cpu/rnn/*.cpp",
         "src/cpu/rnn/*.hpp",
         "src/cpu/xbyak/*.h",
-    ]) + [":mkldnn_version_h"],
+    ]) + if_mkl_v1_open_source_only([
+        "src/cpu/jit_utils/jit_utils.cpp",
+        "src/cpu/jit_utils/jit_utils.hpp",
+    ]) + [":mkldnn_version_h"] + if_mkl_v1_open_source_only([
+        ":mkldnn_config_h",
```

Comment:
Please move this to the previous `if_mkl_v1_open_source_only` (two lines above).

Reply:
@penpornk Done.

Reply:
The list should be sorted alphabetically. Could you please move `:mkldnn_config_h` to above `src/cpu/jit_utils/jit_utils.cpp`? Sorry for the inconvenience!